api/test/wfl/unit/gcs_test.clj

```diff
@@ -57,32 +41,9 @@
       (is (thrown? IllegalArgumentException (gcs/gs-url ""  "")))
       (is (thrown? IllegalArgumentException (gcs/gs-url ""  "o"))))))
 
-(deftest bucket-test
-  (testing "Buckets"
-    (testing "bad names"
-      (let [too-short  (str/join (take 2 (str blame blame)))
-            too-long   (str/join (take 64 (str prefix uid uid)))
-            uppercase  (str prefix "Uppercase")
-            underfirst (str "_" prefix 0)]
-        (is (thrown? IllegalArgumentException (make-bucket too-short)))
-        (is (thrown? IllegalArgumentException (make-bucket too-long)))
-        (is (thrown? IllegalArgumentException (make-bucket uppercase)))
-        (is (thrown? IllegalArgumentException (make-bucket prefix)))
-        (is (thrown? IllegalArgumentException (make-bucket underfirst)))))
-    (let [bucket (first buckets)]
-      (testing "make"
-        (is (= [bucket "US" "STANDARD"]
-               ((juxt :name :location :storageClass)
-                (make-bucket bucket)))))
-      (testing "list project"
-        (is (<= 1 (count (gcs/list-buckets project)))))
-      (testing "list project prefix"
-        (let [result (gcs/list-buckets project prefix)]
-          (is (= 1 (count result)))
-          (is (= bucket (:name (first result))))))
-      (testing "delete"
-        (is (gcs/delete-bucket bucket))
-        (is (not (gcs/delete-bucket bucket)))))))
+(def blame
+  "Who is to blame?"
+  (or (System/getenv "USER") "wfl"))
```

Comment:
do we still need this function?

Reply:
Good point! It's used [here](https://github.com/broadinstitute/wfl/blob/37a92e7abe9b93c000e5717515583df079843064/api/test/wfl/unit/gcs_test.clj#L50) but it's just for the local file name. I can change that file name prefix to "wfl" instead.

Reply:
Oh I see! I think it makes sense to keep it. I just didn't realized it's still being used!

Reply:
I just removed it since it's only used for the local files in the test (previously it was also used as the bucket name so you could tell who created it)

Reply:
Sounds great!